dev-docs/bidders/nobidBidAdapter.md

```diff
@@ -4,12 +4,16 @@ title: Nobid
 description: Prebid Nobid Bidder Adaptor
 biddercode: nobid
 pbjs: true
+pbs: true
 media_types: banner, video
 gdpr_supported: true
+tcf2_supported: true
+gvl_id: 816
 usp_supported: true
 schain_supported: true
 coppa_supported: true
-tcf2_supported: true
+userId: all
```

Comment:
Sorry, "all" isn't a valid value here. There's work that needs to be done to support each of the 18+ user ID modules. Please list them out. e.g. "britepoolId, criteo, digitrust, id5Id, identityLink, liveIntentId, netId, parrableId, pubCommonId, unifiedId"

Reply:
What we do, it that we take the whole user-id object and ship it to the back-end. All we need is the source and the ID and those are standard. I am not sure what would be the difference between user id providers that would change our implementation.